canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -35,6 +35,70 @@ $toolbar-divider-width: 1px;
 
 	/* Prevent elements from being dragged onto canvas */
 	-webkit-user-drag: none;
+	&.toolbar-size-small {
+		.toolbar-left-bar {
+			align-items: center;
+			text-align: center;
+			padding: 0px;
+		}
+		.toolbar-item {
+			&.default,
+			&.ghost {
+				& button {
+					padding: 3px;
```

Comment:
Why is this only for default and ghost types? 
The button is wrong. Ypu can see the height extends past the toolbar 
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/1628100/199132094-f6de8658-3c7d-4804-aa7f-221fec21e8cc.png">
<img width="204" alt="image" src="https://user-images.githubusercontent.com/1628100/199132175-106102f0-96ae-4bce-a1a2-0dfab88628b3.png">

Reply:
<img width="804" alt="image" src="https://user-images.githubusercontent.com/89609176/199339973-fd976710-86df-43af-bc32-1a37a2f03e5f.png">